fab/fabfile.py

```diff
@@ -218,18 +218,18 @@ def _setup_env(env_name, force=False, default_branch=None):
 
 
 def _confirm_branch(default_branch=None):
-    if env.code_branch != 'master':
+    if env.code_branch == 'master':
         if default_branch:
             env.code_branch = default_branch
```

Comment:
Seems odd that it's impossible to deploy `master` to staging. Looks like it's been that way for a long time, even before 390d47e810f5fd2d81980bb60a7cef0893721229, so I'm not saying it needs to change.